.github/workflows/ci.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
           - charm-integration
           - database-relation-integration
           - db-relation-integration
-          - db-admin-relation-integration
+#          - db-admin-relation-integration
```

Comment:
Might be better to mark it as `unstable`

Reply:
I tried with it before, but the test failed. I think it's because there is only one test in that file.

Reply:
@carlcsaposs-canonical can you please check this case and improve unstable concept... commenting tests makes them invisible. 

@marceloneppel please create a bugreport ticket to re-enable the test. So we can schedule it in a pulse.

Reply:
Created: https://warthogs.atlassian.net/browse/DPE-1414